rust/src/sip/parser.rs

```diff
@@ -50,6 +50,7 @@ pub struct Response {
     pub version: String,
     pub code: String,
     pub reason: String,
+    pub headers: HashMap<String, String>,
```

Comment:
What happens if I have multiple `via` headers ?

Reply:
The value is updated and the old one is returned.